app/serializers/workflow_serializer.rb

```diff
@@ -25,13 +25,17 @@ def self.links
   end
 
   def version
-    "#{ModelVersion.version_number(@model)}.#{ModelVersion.version_number(content)}"
+    "#{@model.current_version_number}.#{content_version}"
   end
 
   def content_language
     content.language if content
   end
 
+  def content_version
+    content.try(:current_version_number) || ModelVersion.default_version_num
```

Comment:
This is the main bit I'd like to see refactored/cleaner, but didn't have a good idea how. If nobody else does, let's just leave it, but have a think.